### DISCUSSION TITLE
Could this package be used by pure react app without nextjs?

### DISCUSSION THREAD

**Discussion by yatongzhao0622** [47ng/nuqs#598] [Q&A] [answered]
I tried to use this package in my project where no nextjs present, but I got errors like `Module not found: Error: Can't resolve 'next/navigation.js'`

**Comment by franky47** [comment, marked as answer]
In its present form, it is for Next.js only.

What's your React app setup like? Are you using a router?

**Reply by yatongzhao0622** [reply]
I'm using react router v5.
